### ISSUE TITLE
Default values on @QueryParam

### ISSUE THREAD

**Issue by choffa** [papsign/Ktor-OpenAPI-Generator#118]
Would it be possible to support default values for optional query params? Either as a value in the `@QueryParam` annotation, or by using a default value from the constructor?

**Comment by Wicpar** [comment]
the obvious way to do it if you are willing to do a one liner is to do something like this, with `params.param` as a nullable type:
`val param = params.param ?: getDefaultValue()`

**Comment by choffa** [comment]
Yes, that is what I am doing today. I was more interested in knowing if there is a way to set the `default` field in the schema, as described [here](https://swagger.io/docs/specification/describing-parameters/#default) or if you would be interested in implementing such a feature :)

**Comment by Wicpar** [comment]
If you want to do a pr to add it i would accept it :)

**Comment by HuiiBuh** [comment]
I am really interested in contributing (especially #122), but I am having some problems understanding the setup.

Where exactly does the collection of the schemas happen and how do they get collected. It does not seem to be near the route functions which handle the requests.

I think I am looking for some basic description which describes the work flow involved in creating the OpenAPI file
